gapic/src/main/com/google/gapid/perfetto/models/CpuSummaryTrack.java

```diff
@@ -36,7 +36,7 @@
       "select quantum_ts, sum(dur)/cast(%d * %d as float) " +
       "from %s where utid != 0 group by quantum_ts";
   private static final String SLICE_RANGE_SQL =
-      "select row_id, ts, dur, cpu, utid, upid, end_state, priority " +
+      "select id, ts, dur, cpu, utid, upid, end_state, priority " +
```

Comment:
I think it should be sched.id here, there would be "ambiguous column name: id" exception at area-selection in this panel. (This cpu summary panel appears when collapsing 8 cpus into one)/

Reply:
Done